### DISCUSSION TITLE
valorant crashing or freezing when using game capture obs

### DISCUSSION THREAD

**Discussion by hamptersanpaiTTV** [obsproject/obs-studio#11661] [General]
ive had this issue for a few days and the only way i found to fix it was to download a older version of obs 30.2.3 is what im using now and have 0 issues  hope this helps someone ♥

**Comment by mihawk90** [comment]
Riot needs to update their anticheat as per [the announcement](https://obsproject.com/kb/capture-hook-certificate-update).

In the meantime use window capture.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Please report this to Valorant/Riot.

Thank you!
